### DISCUSSION TITLE
documentation load performance

### DISCUSSION THREAD

**Discussion by acivitillo** [lowdefy/lowdefy#1017] [General]
Lowdefy documentation can be quite slow to load, I find myself starting a tab and then do something else while it loads often. Currently docs.lowdefy.com can take a couple of minutes to load.

**Comment by Gervwyk** [comment]
Hi @acivitillo 

Thanks for the feedback. Sorry that it is this slow, for me it never takes more than a few seconds to load. 

The good news is we have already made significant performance improvements on the v4 branch. In v4 we will also be supporting static sites / pages. So once we publish the v4 docs it should be as fast as a regular next.js app. v4 is already in alpha release and you can expect the new docs sometime in February.

**Reply by acivitillo** [reply]
Amazing thanks. I am very happy with Lowdefy, I am giving you this feedback because I thought maybe you could not be aware of the loading times yet.
